src/__tests__/imagings/util/validate-imaging-request.test.ts

```diff
@@ -0,0 +1,17 @@
+import validateImagingRequest from '../../../imagings/util/validate-imaging-request'
+import Imaging from '../../../shared/model/Imaging'
+
+describe('imaging request validator', () => {
+  it('should validate the required fields apart of an incident', async () => {
```

Comment:
Not a train smash, but may we fix test description, "are part of an incident" instead of "apart of an incident"?